packages/data/addon/adapters/dimensions/bard.js

```diff
@@ -106,10 +106,23 @@ export default class BardDimensionAdapter extends EmberObject {
       andQueries = [andQueries]; // if not array, wrap
     }
     assert("You must pass an 'Array' of queries to be ANDed together", Array.isArray(andQueries));
+
+    const defaultDimensionField = this._getDimensionMetadata(
+      dimension,
+      options.dataSourceName || getDefaultDataSourceName()
+    ).primaryKeyFieldName;
+
+    andQueries.forEach(query => {
+      if (!query.field?.includes('.')) {
+        // field passed in is typically id/desc, but we want it to be 'dimension.dimensionField'
```

Comment:
Is this temporary? As we are looking to model this as a param?

Reply:
Yes, right now bard-facts expects it to be in the name and not a parameter
https://github.com/yahoo/navi/blob/dd8c6bbd95397fe94af3244042fec23723169644/packages/data/addon/adapters/bard-facts.ts#L34-L35